_posts/en/newsletters/2022-09-07-newsletter.md

```diff
@@ -55,7 +55,13 @@ prevent an attacker sending the main chain during presync and then a
 different, malicious chain during redownload, the node stores
 commitments to the headers chain during presync.
 
-- [Bitcoin Core #25355][] I2P: add support for transient addresses for outbound connections FIXME:Xekyo
+- [Bitcoin Core #25355][] adds support for transient, one-time I2P
+  addresses. In I2P, the recipient learns the I2P address of the
+  connection initiator. Nodes using the new configuration option
+  `-i2ptransientout` use a new transient I2P address for every initiated
```

Comment:
The approach in 25355 got changed during the review process  and unfortunately it was forgotten to update the OP accordingly - see the release note in https://github.com/bitcoin/bitcoin/pull/25355/commits/d7ec30b648721133b5a5ac3f52275f779c54310f for what happens:
There is no longer a new `-i2ptransientout` option, the logic that was merged is that we make our outbound connections transient only if inbound i2p connections are disallowed (`-i2pacceptincoming=0`). Otherwise, when we want incoming connections via i2p, we need a stable identity for that and therefore keep on using and advertising a non-transient address as before.

I also asked in https://github.com/bitcoin/bitcoin/pull/25355#issuecomment-1238725395 to update the PR description.